docs/administration/faculty-adviser.rst

```diff
@@ -1,53 +1,62 @@
-Faculty Adviser
-===============
+###############
+Faculty advisor
+###############
 
-All official RIT clubs must have a faculty adviser.
-The faculty adviser is a more permanent position in the club and they are not subject to annual elections.
+This document explains the role of the faculty advisor in RITlug and how to on-board and off-board a faculty advisor.
 
-The faculty adviser may change at any time during the year.
-If this happens, a new adviser should be found immediately (or else the club risks losing recognition by student government).
-Once found, the former adviser should be off-boarded and the new one on-boarded.
 
+*********************
+About faculty advisor
+*********************
 
-Adviser responsibilities
-------------------------
+All recognized RIT clubs must have a faculty advisor.
+The faculty advisor is a permanent position and not subject to annual elections.
 
-Faculty advisers can take a hands-on or hands-off role.
-In the past, RITlug's adviser is usually hands-off, assuming the club is well-behaved and there is regular communication with the adviser.
-If a new adviser is selected, they should meet with the current eboard and discuss expectations.
+Responsibilities
+================
 
-The adviser is a point of contact for the club and RIT clubs office.
-Thus, there are a few forms that the adviser must sign annually (usually at the beginning and end of the year).
-Periodically, the adviser may be needed to resolve issues with the clubs office or with renewing club resources.
+Faculty advisors can take a hands-on or hands-off role.
+Historically, RITlug's advisor is hands-off.
+This assumes the club is in good standing is regular communication with the advisor.
```

Comment:
Wording is odd with this sentence.

Reply:
@Tjzabel Oops, nice catch. I just pushed a new commit to clean this up.